contracts/staking/ValidatorManager.sol

```diff
@@ -324,6 +325,8 @@ abstract contract ValidatorManager is
 
         // Emit event.
         emit ValidationPeriodEnded(validationID, validator.status);
+
+        return validator;
```

Comment:
I know this isn't part of your PR, but it makes no sense to emit `validator.status` on line 327 because it will always be `endStatus`

Reply:
the status can be completed or invalidated

Reply:
Ohhh I see. We don't actually even need the `endStatus` variable, we should just set `validator.status` directly, since we're reading from it anyway.